### PR TITLE
Use terraform fmt for canonical terraform formatting

### PR DESCRIPTION
**Pull request by iantivey**
`terraform fmt` is the canonical method for formatting Terraform for readability.  I suggest we require the use of this before any PRs are accepted into the project.

### CODE REVIEW

**Comment by mcleo-d** [comment]
Thank you @iantivey for raising this PR. I have assigned to @peterrhysthomas and am having conversations with @mindthegab regarding the Citihub CCLA, which will clear the CLABot warning.

I'll add this PR to the next CSC meeting for you to introduce to the group. I hope that works?

James.

**Review by abdullahgarcia — APPROVED** [review]
Looks good to me.

**Comment by TLATER** [comment]
At risk of shaving yaks, maybe CI to actually run this on PRs should also be included *with this PR*. A GitHub action exclusively for formatting shouldn't be terrifyingly complex, even if the CI will later be extended with probr & co.

Otherwise it could make sense to postpone it until that lands; the cost of merging this PR won't increase since it's just autoformatting.

Slip-ups happen quickly, and reviewers probably won't pull PRs just to run `terraform fmt` on them, so IME having CI to check this is very important to stay consistent. There's also not much value to autoformatting unless it stays consistent (since it will just keep introducing whitespace changes when a mix of people using/not using it exists, often resulting in overhead on PRs).

**Comment by TLATER** [comment]
Separate point from that - nit: The two commits should probably be squashed.

File: gcp/gke/gke-terraform-scripts/modules/kubernetes/main.tf [hunk 1]
```diff
@@ -79,7 +79,6 @@ resource "google_container_node_pool" "node-pool-1" {
     max_unavailable = 0
   }
 
-
```
Comment (TLATER) [inline gcp/gke/gke-terraform-scripts/modules/kubernetes/main.tf:82]: Nit: This should probably be squashed into the other commit.

**Comment by mcleo-d** [comment]
Hi @peterrhysthomas 👋 

As per the team discussions during #114 can you review, accept and merge this pull request for @iantivey and @eddie-knight.

Many thanks,

James 🚀

**Review by peterrhysthomas — APPROVED** [review]
Looks good to me, separate issue raised for automating the format check

**Pull request merged**